### DISCUSSION TITLE
Category chart missing for 1 specific category

### DISCUSSION THREAD

**Discussion by psleep** [firefly-iii/firefly-iii#6576] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I am very new to firefly iii but have managed to import my data from both bank accounts from January to date, set up my liabilities and bills accounts as well as a bunch of categories - it really is an _amazing_ product!

One of my categories however, (Classification > Categories) does not display any charts when all of my other categories do., despite having nearly 300 transactions listed. 

### Expected behaviour

Charts to be displayed for seleted time period and year to date, 

### Debug information

Debug information generated at 2022-10-26 21:47:37 UTC for Firefly III version **5.7.14**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.7.14 |
| Firefly III API | 1.5.6 |
| Build | 286, 18-10-2022 21:16:12 CEST |
| DB version | 18 (exp. 18) |
| PHP | `8.1.11` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ |  |
| Browser TZ | Europe/London |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | info, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `**` |

| User info | Value |

### Steps to reproduce

1. Select Classification > Categories
2. Choose a category (in this particular case mine is "Savings")
3. Chart does not show

**Comment by JC5** [comment]
If it's just a single chart, it's probably a cache issue. Browse to `/flush` and try again.

**Reply by psleep** [reply]
Hi JC5,
Firstly thanks for the lightning fast response!
For that alone I will probably end up subscribing to your patreon in the new year (gives me a couple of months to really try Firefly III).

I did a flush but that didn't seem to help.

But I have realised that the difference is all of the transactions for the Savings category are transfers rather than withdrawal or expenses as I am also tracking the savings account in Firefly iii

Is this expected behaviour? 
Would you recommend I change my inputs to a withdrawal from my main account and then another transaction as deposit to my savings account?

Thanks

**Reply by JC5** [reply]
Thanks! There should be an error log somewhere, either in storage/logs or by tracking your Docker container.

> Is this expected behaviour?

Yes, transactions between two asset accounts (savings accounts are also asset accounts) are always transfers.

**Reply by psleep** [reply]
Sorry, what I meant by 'expected behaviour' is that _should_  transfers appear in that category chart or not?

I also have another category called 'Transfers' and the only transaction that shows in the chart for that one is a deposit so I don't think transactions that transfer amounts between asset (or savings) accounts show up in the category charts - do you think that is normal?
Thanks,

Phil

**Reply by JC5** [reply]
Ah yes, transfers do not show up in these charts. I can't remember if I intended it to be so or not. It would skew the charts, probably. I'm inclined to leave them out, but I'll give it some thought :+1:

**Reply by psleep** [reply]
No worries - just wanted to make sure it wasn't something I had managed to cause to break I can easily track the balance of the savings account via reports anyway.

You've done such a great job building this tool - it's going to completely change my financial health - just being able to see all of my accounts and liabilities together with one overview is so useful - add in the fact that I will be manually adding transactions throughout the month I think it's going to be a game changer for me :)

**Reply by JC5** [reply]
Woohoo! Glad to hear you like the tool 😁
